core/domain/suggestion_services.py

```diff
@@ -593,12 +616,213 @@ def get_translation_suggestions_waiting_longest_for_review_per_lang(
     return [
         get_suggestion_from_model(suggestion_model) for suggestion_model in (
             suggestion_models.GeneralSuggestionModel
-            .get_translation_suggestions_waiting_longest_for_review_per_lang(
+            .get_translation_suggestions_waiting_longest_for_review(
                 language_code)
         )
     ]
 
 
+def _get_plain_text_from_html_content_string(html_content_string):
+    """Retrieves the plain text from the given html content string. RTE element
+    occurrences in the html are replaced by their corresponding rte name,
+    capitalized in square brackets.
+    eg: <p>Sample1 <oppia-noninteractive-math></oppia-noninteractive-math>
+        Sample2 </p> will give as output: Sample1 [Math] Sample2.
+    Note: similar logic exists in the frontend in format-rte-preview.filter.ts.
+
+    Args:
+        html_content_string: str. The content html string to convert to plain
+            text.
+
+    Returns:
+        str. The plain text string from the given html content string.
+    """
+
+    def _replace_rte_tag(rte_tag):
+        """Replaces all of the <oppia-noninteractive-**> tags with their
+        corresponding rte name in square brackets.
+
+        Args:
+            rte_tag: MatchObject. A matched object that contins the
+                oppia-noninteractive rte tags.
+
+        Returns:
+            str. The string to replace the rte tags with.
+        """
+        # Convert the MatchObject to a string.
```

Comment:
This is not doing a conversion, it is obtaining the "matched" string from the matched object. Please clarify.

Reply:
Looks resolved.